lib/Echidna/Campaign.hs

```diff
@@ -281,11 +283,12 @@ campaign u v w ts d txs = do
   where
     step        = runUpdate (updateTest w v Nothing) >> lift u >> runCampaign
     runCampaign = use (hasLens . tests . to (fmap snd)) >>= update
+    bsm         = makeBytecodeMemo . mapMaybe (viewBuffer . (^. bytecode)) . elems $ (v ^. env . EVM.contracts)
     update c    = do
       CampaignConf tl sof _ q sl _ _ _ _ _ <- view hasLens
       Campaign { _ncallseqs } <- view hasLens <$> get
       if | sof && any (\case Solved _ -> True; Failed _ -> True; _ -> False) c -> lift u
-         | any (\case Open  n   -> n < tl; _ -> False) c                       -> callseq v w q >> step
+         | any (\case Open  n   -> n < tl; _ -> False) c                       -> callseq bsm v w q >> step
```

Comment:
Maybe this should be put inside the Campaign instead of passing it explicitly? Not sure though